modules/ROOT/pages/customize.adoc

```diff
@@ -0,0 +1,107 @@
+= Customize the Look and Feel of Your API Community
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Use the https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] to customize the branding, look and feel, and site structure of your community pages to match your desired brand identity.
+
+Select *Community Builder* in the API Community Manager (ACM) administrator interface. By default, the
+Community Builder displays the *Home* page and the Pages drop-down list displays *Home*. You can select
+the Pages drop-down list to display a list of community pages and specify a different page for customization.
+
+The *Community Builder* interface contains the customization icons described here.
+
+== Components
+
+Select the *Components* lightning bolt icon in the top left navigation to display the component palette.
+The component palette includes Salesforce Community Cloud and ACM-specific components.
```

Comment:
"...components, such as ...." Suggest naming a couple here, just to give the audience an idea of what these are.

Reply:
Good idea! Added these.